### ISSUE TITLE
ImportError: cannot import name 'leading_empty_lines'

### ISSUE THREAD

**Issue by ADutchTourist** [spyder-ide/spyder-kernels#256]
After upgrading to Spyder 4.2.0 i receive the following error.

```python-traceback
An error ocurred while starting the kernel
Traceback (most recent call last):
File "c:\python37\lib\runpy.py", line 193, in _run_module_as_main
"__main__", mod_spec)
File "c:\python37\lib\runpy.py", line 85, in _run_code
exec(code, run_globals)
File "c:\python37\lib\site‑packages\spyder_kernels\console\__main__.py", line 23, in 
start.main()
File "c:\python37\lib\site‑packages\spyder_kernels\console\start.py", line 253, in main
import_spydercustomize()
File "c:\python37\lib\site‑packages\spyder_kernels\console\start.py", line 43, in import_spydercustomize
import spydercustomize
File "c:\python37\lib\site‑packages\spyder_kernels\customize\spydercustomize.py", line 36, in 
from IPython.core.inputtransformer2 import (
ImportError: cannot import name 'leading_empty_lines' from 'IPython.core.inputtransformer2' (c:\python37\lib\site‑packages\IPython\core\inputtransformer2.py)
```

![Spyder kernel](https://user-images.githubusercontent.com/57393627/98547639-c62ad000-2298-11eb-8792-58f5696b4f37.JPG)

**Comment by ccordoba12** [comment]
Hey @ADutchTourist, thanks for reporting. Could you let us know what's your IPython version?

**Comment by ADutchTourist** [comment]
Hello Carlos, this concerns the latest version number 4.2.0. where the problem exists.
I downgraded back to 4.1.6. to keep on working with this amazing IDE.

**Comment by Yorkshire-Jack** [comment]
I am getting the same error.  My ipython=7.4.0  spyder-kernels=1.10.0 and spyder=4.2.0.

I can't quite tell from the thread if this is a known error or if you need more information. Happy to answer questions.

**Comment by ccordoba12** [comment]
It is a known error. We'll fix it in our next version (1.10.1), to be released in three weeks.

In the meantime, please update IPython to its latest version (7.19.0)
